packages/teuchos/comm/src/Teuchos_StackedTimer.cpp

```diff
@@ -255,6 +283,20 @@ StackedTimer::computeColumnWidthsForAligment(std::string prefix,
           os << "}";
         alignments_.max_ = std::max(alignments_.max_,os.str().size());
       }
+      if (procmin_.size()) {
+        std::ostringstream os;
+        os << ", proc min=" << procmin_[i];
+        if (active_[i] <= 1)
+          os << "}";
+        alignments_.procmin_ = std::min(alignments_.procmin_,os.str().size());
```

Comment:
The two arguments of `min` might have different type here; that could lead to warnings at best.